explainer.md

```diff
@@ -27,6 +27,9 @@ footprint, thus making this task very difficult.
 See [Appendix D](#appendix-d) for a description of Chrome's current
 implementation of performance.memory, and why it does not solve the problem.
 
+Moreover performance.memory in Chrome exposes jsHeapSizeLimit which is an JavaScript engine specific
```

Comment:
can we move this into Appendix D? 

tdresser pointed out that many of the viewers won't be interested in Chrome's previous implementation, and thus I moved it all into Appendix D. I think we can just move this line and everything will still read fine.